testcases/crypto/ec_func.c

```diff
@@ -1081,6 +1082,7 @@ CK_RV run_DeriveECDHKeyKAT(void)
             {CKA_KEY_TYPE, &secret_key_type, sizeof(secret_key_type)},
             {CKA_VALUE_LEN, &(ecdh_tv[i].derived_key_len), sizeof(CK_ULONG)},
             {CKA_SENSITIVE, &false, sizeof(false)},
+            {CKA_IBM_PROTKEY_EXTRACTABLE, &true, sizeof(true)},
```

Comment:
Shouldn't we make this dependent on the pkey mode here as well? Also add CKA_EXTRACTABLE to the template, otherwise this might not even have an effect for EP11.

Reply:
Done.